src/styles/_header.scss

```
@@ -37,5 +37,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition-duration: 300ms;
```

```suggestion
    transition-duration: 300ms;
    transition-property: color
```
Завжди потрібно вказувати, до чого саме застосовується transition
https://developer.mozilla.org/en-US/docs/Web/CSS/transition-property